src/api/app/assets/stylesheets/webui2/texts.scss

```diff
@@ -0,0 +1,7 @@
+.text-package {
+    color: $obs_yellow;
+}
+
+.text-gray-550 {
```

Comment:
There are [color classes](https://getbootstrap.com/docs/4.1/utilities/colors/) for that...

Reply:
The scale of grey that bootstrap offers us doesn't fit well, so we add more.